namelayer-spigot/src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: NameLayer
 main: vg.civcraft.mc.namelayer.NameLayerPlugin
-version: ${project.version}
+version: ${releaseVersion}
```

Comment:
So project.version binds to the "project.version" that is within the pom controlling the filtering.

We recently changed all those poms to be "fixed" at version 1.0.0. So all the plugin/bungee.yml files had version 1.0.0 in them instead of the real release version, which made the results of /version Namelayer in game doubling confusing, especially since our release tooling strips versions from the file name as well. Literally no way to find out version without searching build for a hash...